revisitedTBAGame.js

```diff
@@ -1,151 +1,172 @@
-//       for (let i = 0; i < num; i++) {
-//         callback(num)
-//       }
-//       return callback;
-//     }
-//     bangManyTimes(3, (callback) => console.log("BANG!"));
-    
-//     readyOrNot = readline.question("HOORAY! You did it! You made truffles! Are you ready to reap the rewards??")
-//     if (readyOrNot === "yes") {
-//       console.log("Hell yeahhhhh")
-//     } else {
-//       console.log("Shucks, looks like I'll have to eat it all myself ;)")
-//     }
-    
-//     let chocolateTime = readline.question("CUE THE FANFAIR! The moment of truth---It's chocolate time! Enter to fall in love <3");
-    
-//     let numInput;
-//     numInput= 1 < 10
-//     numInput = readline.question("How many pieces? Pick a number from 0-10");
-//     switch (numInput) {
-//       case '10':
-//         console.log("Wow! You're a chocoholic like me! Enjoy! Come back for more chocolate making any time!! Thank you for playing my game!");
-//         break;
-//         case '9':
-//           case '8':
-//             case '7':
-//               case '6':
-//                 console.log("Cool! You have enough for sharing--or not! Come back for more! Thanks for playing!" );
-//                 break;
-//                 case '5':
-//                   case '4':
-//                     case '3':
-//                       case '2':
-//                         case '1':
-//                           console.log("That's great, just enough to tantilize ya! I hope you come back for more! Thanks for playing!" );
-//                           break;
-//                           default:
-//                             console.log("That's too bad! Come back when you change your mind!");
-                        
-// }
-//    }
-}
\ No newline at end of file
+      console.log(
+        "You've never had mine! Talk to me later on for some samples ;)"
+      )
+      userInput = readlineSync.question("Do you like chocolate?")
+    }
+    if (userInput === "yes");
+    {
+      console.log(
+        "Awesome! Let's get ready to make some truffles! I'll show you how! Just help me with some measuring!"
+      )
+    }
+
+    let readySetGo = readlineSync.question(
+      "Let's get started, shall we? Enter and we shall!"
+    )
+    let pickOne = readlineSync.question(
+      "First, pick your favorite! What's it gonna be?! Sweet, smooth, and creamy MILK CHOCOLATE |or| rich, robust, and decadent DARK CHOCOLATE? Enter milk or dark!"
+    )
+    // whichOne() {
+    let which = userInput
+    if (pickOne === "milk") {
+      console.log("Sweet! Milk chocolate it is!")
+    } else {
+      console.log("Nice! Dark chocolate for you!")
+    }
+
+    let gramsToOz = readlineSync.question(
+      "We have to measure how much chocolate we'll need to melt! We want a big batch, so lets's go for 24 ounces! Would you help convert 24 ounces to grams? Just get the product of the mass value (24 oz.) by 28.35!"
+    )
+
+    let arrGrams = [24, 28.35]
```

Comment:
This is all good,  but you should look into array destructuring. In this scenario, this can help you add variable names to array elements. such as this.

`let minMax = [10,2010]

let [min, max] = minMax
console.log(min)`

If you need more explanation on this  don't be shy to ask Jhenya or me.